Sources/Brave/Frontend/Browser/BrowserViewController/BVC+Rewards.swift

```diff
@@ -29,14 +29,6 @@ extension BrowserViewController {
     self.topToolbar.locationView.rewardsButton.iconState = Preferences.Rewards.rewardsToggledOnce.value ? (rewards.isEnabled || rewards.isCreatingWallet ? .enabled : .disabled) : .initial
   }
 
-  func showRewardsDebugSettings() {
```

Comment:
We are ok with removing this interaction ?

Reply:
No need to duplicate QA functionality, it's 1 tap away from the settings screen and our Ads/Rewards usage on iOS is light.
What do you think @kylehickinson

Reply:
Its technically been broken ever since we added the context menu anyways, so fine to remove it IMO

Reply:
yeah. we will readd if needed. also as a contextmenu action to mvoe away from gesture recognizers